deliverables/best_practices.md

```diff
@@ -125,3 +125,106 @@ Always choose the version of Python that is correct for your usage.
   The `typing` module was new in Python 3.5
   but additional features from the Python 3.8 version are also backported.
   See https://docs.python.org/3.8/library/typing.html.
+
+## Avoid use of futurize stage 2 and its standard library backports
+
+[futurize](https://python-future.org/overview.html#automatic-conversion-to-py2-3-compatible-code)
+can be helpful for transitioning standalone projects and scripts to
+Python 3, but `--futurize --stage2` adds dependencies on the
+`future.standard_library` module that changes the behavior of the Python
+standard library.  This makes it unsuitable for use in libraries or plugins
+that have to co-exist alongside other python plugins in a host application.
+
+It is preferrable to use `six` and the standard library instead of the
+`future`, `past`, and `builtins` modules provided by `python-future`.
+
+## Read the Conservative Python 3 Porting Guide
+
+[The Conservative Python 3 Porting Guide](https://portingguide.readthedocs.io/en/latest/)
+(CPPG) is an invaluable resource for porting to Python 3.
+This document borrows heavily from the CPPG recommendations.
+
+## Read the futurize Python 3 Compatibility Cheat Sheet
+
+The futurize project contains a
+[Cheat Sheet](https://python-future.org/compatible_idioms.html)
```

Comment:
This guide and the above porting guide cover the same thing for the most part. Perhaps instead of recommending reading both we suggest one or the other depending on the situation?
The porting guide is much better for reading and learning from I think, whereas the future guide is a simple list of everything and is more useful for reference (eg linking to and doing a quick find).

Reply:
That's a good point.  Maybe adding the futurize docs to the mix here is actually making things too complicated since we would then have to explain the situation where the futurize docs may or may not be appropriate.

It seems like the simplest thing would be to recommend the CPPG and maybe link to the futurize cheat sheet in a separate "Additional Links" section at the end.  That way we acknowledge it, but our main recommendation becomes the CPPG.

Reply:
That sounds like a good approach to me. I like it!